### DISCUSSION TITLE
Overfitting in the IEMOCAP wav2vec2 model

### DISCUSSION THREAD

**Discussion by Markweii** [speechbrain/speechbrain#1935] [Q&A] [answered]
![000](https://user-images.githubusercontent.com/118371061/231165492-c2fa9795-7fa0-41c8-970b-c320fa931a8b.png)
Hey guys! I have a question.
In general, if the valid loss continues to increase, it indicates overfitting. However, why is it that while the valid loss continues to increase, the test error rate can continue to decrease?

**Comment by BenoitWang** [comment, marked as answer]
Hi @Markweii , I found a good explanation [here](https://stats.stackexchange.com/questions/282160/how-is-it-possible-that-validation-loss-is-increasing-while-validation-accuracy). Briefly, the loss and the accuracy are not 100% inversely correlated. When the outputs change, they change directly the loss, but they need to go over a threshold to change the accuracy.

I found also an image that shows the case, as you can see it's all about the output distribution.
![gppxn](https://user-images.githubusercontent.com/41187612/231463803-718d86a2-25c3-4dfe-bad4-04023f37629e.png)

**Reply by Markweii** [reply]
Hi @BenoitWang ,
I got it, thanks a lot for the help!
